vss-accessor/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+use ::prost::Message;
+use reqwest;
+use reqwest::Client;
+use std::error::Error;
+
+use crate::vss::{GetObjectRequest, GetObjectResponse, KeyValue, ListKeyVersionsRequest, ListKeyVersionsResponse, PutObjectRequest, PutObjectResponse};
+use crate::vss_error::VssError;
+
+mod vss_error;
+
+pub mod vss {
+    include!(concat!(env!("OUT_DIR"), "/org.vss.rs"));
+}
+
+pub struct VssAccessor {
```

Comment:
This needs docs as do all the `pub` methods below.

Reply:
I would like to up take the documentation task separately.
At high level it should be similar to proto docs.

Reply:
Why? Documentation is an important part of code review. We don't separate that elsewhere in LDK and shouldn't get into the habit of it now.

Reply:
Its different in LDK because we have a release schedule and we don't want code to go to clients without docs. 

Here i just want to keep meaningful sizeable chunks of work separate and decouple them.
Nobody is saying client is "complete" without docs. 
Can create an issue for it if you want. Helps me keep work-parts separate and not create churn in one PR.

Reply:
As someone reviewing the code, lack of docs make it more difficult. Please add them in this PR.

Reply:
Added 3-liner docs for now which mainly refer to proto generated docs for more info.